plugins/cpu/fu-cpu-device.c

```diff
@@ -136,22 +143,22 @@ fu_cpu_device_add_instance_ids(FuDevice *device, GError **error)
 	if (!fu_cpuid(0x1, &eax, NULL, NULL, NULL, error))
 		return FALSE;
 	processor_id = (eax >> 12) & 0x3;
-	model_id = (eax >> 4) & 0xf;
+	self->model_id = (eax >> 4) & 0xf;
 	family_id = (eax >> 8) & 0xf;
```

Comment:
considering my comment about the affected being only `0x17` and `0x19`, maybe store `family_id` for the lookup later too.